regression/goto-instrument/nondet-volatile-model-03/test.c

```diff
@@ -0,0 +1,20 @@
+#include <assert.h>
+
+volatile int a;
+volatile int b;
+
+int model_a()
+{
+  return 1;
+}
+
+int model_b()
+{
+  return 2;
+}
+
+void main()
+{
+  int result = a + b;
```

Comment:
Yes that's right, it currently it only works for direct variable reads.

(One could also mark a pointer as nondet, in which case the pointer derefernces would be nondet as well. Though when `--pointer-check` would be enabled it would fail.)